src/code/z_bg_collect.c

```diff
@@ -90,12 +90,12 @@ u32 BgCheck2_UpdateActorAttachedToMesh(CollisionContext* colCtx, s32 index, Acto
         return 0;
     }
 
-    if ((meshActor->unk154 & 1) != 0) {
+    if ((meshActor->flags & 1) != 0) {
         BgCheck2_UpdateActorPosition(colCtx, index, actor);
         wasUpdated = 1;
     }
 
-    if ((meshActor->unk154 & 2) != 0) {
+    if ((meshActor->flags & 2) != 0) {
```

Comment:
Same here
```suggestion
    if (meshActor->flags & 2) {
```

Reply:
Technically out of scope as the only changes to this file were automated. Updated anyway. z_bg_collect will need a documentation pass eventually.